paywall/src/data-iframe/paywallConfig.js

```diff
@@ -0,0 +1,20 @@
+let paywallConfig
+let paywallConfigLocks
+
+// these are used by the postOffice to filter the locks
+export function setPaywallConfig(newConfig) {
```

Comment:
I'd suggest making this into a class so that you can easily manage the state and reason about things. It may not be called this way currently, but I think someone could call getPaywallConfig without it being set and the same for getRelevantLocks

Reply:
#3789